app/views/main_content_panel.py

```diff
@@ -1761,6 +1761,8 @@ def _do_save(self) -> None:
                         active_mods.append(package_id + "_steam")
                         continue  # Append `_steam` suffix if Steam mod, continue to next mod
                 active_mods.append(package_id)
+        active_mods_uuids, inactive_mods_uuids, _, _ = metadata.get_mods_from_list(mod_list=[mod for mod in active_mods])
```

Comment:
Have not been able to test on my own, but is it necessary to make a copy of the active_mods list?

Reply:
No it is not. Great catch.